libraries/radpdfprocessing/formats-and-conversion/pdf/pdf.md

```diff
@@ -1,26 +0,0 @@
----
```

Comment:
I don't see a lot of added value in this document. The average quality score of this document for the last 6 months is very low - 25%. My suggestion is to remove it. I replaced all links to this document with links to [Using PdfFormatProvider]({%slug radpdfprocessing-formats-and-conversion-pdf-pdfformatprovider%}).